seeds/dex/aggregator/dex_aggregator_seed.csv

```diff
@@ -53,4 +53,7 @@ ethereum,1inch Limit Order Protocol,2,2023-01-14,0x41c3ba345bd8cb15618bb552ad028
 ethereum,1inch Limit Order Protocol,3,2023-01-14,0x0446ab4f7a9a4d765f928db00e4beb28bd5aa199e9b88bd527cc2ba28948ecdb,-1,"1, 0",0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2,0.107251774396181,0xa8b919680258d369114910511cc87595aec0be6d,17.9896
 ethereum,1inch Limit Order Protocol,3,2023-01-14,0x05ac50d8058e90417c85e0cd87508722ccaae23c8249e319f6ae5cffda8107b2,-1,"1, 0",0xdd974d5c2e2928dea5f71b9825b8b646686bd200,682.810765065945,0xdac17f958d2ee523a2206206994597c13d831ec7,450
 ethereum,1inch,UNI v3,2022-07-30,0xb07db4e27b69f65568026181ffba85a37c93baebd23812332232b18f30abad2a,-1,"1, 0, 1",0xeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee,0.0198255454655366,0xae78736cd615f374d3085123a210448e74fc6393,0.0194817986885722
-ethereum,1inch,UNI v3,2022-11-12,0xb07dcf3cf82bb1dd1f9ab6da1562b1af01297e39c9a412a040e9570a67dbd833,-1,,0x9aab071b4129b083b01cb5a0cb513ce7eca26fa5,10053.0060517593,0xdac17f958d2ee523a2206206994597c13d831ec7,2396.746429
\ No newline at end of file
+ethereum,1inch,UNI v3,2022-11-12,0xb07dcf3cf82bb1dd1f9ab6da1562b1af01297e39c9a412a040e9570a67dbd833,-1,,0x9aab071b4129b083b01cb5a0cb513ce7eca26fa5,10053.0060517593,0xdac17f958d2ee523a2206206994597c13d831ec7,2396.746429
+polygon,bebop,1,2023-04-07,0x3107f005f3069e340d7316a5a6372cce5e0b2ba3c42eedc1f4e08e29903620c2,194,,0x7ceb23fd6bc0add59e62ac25578270cff1b9f619,0.16149840785059086,0x2791bca1f2de4661ed88a30c99a7a9449aa84174,300
+arbitrum,bebop,1,2023-04-07,0xeec27114877fa983d5c8cbc9a5cb55dd966ad35e0d35ce30394ed8cd03faee12,12,,0xfd086bc7cd5c481dcc9c85ebe478a1c0b69fcbb9,152.13967,0x539bde0d7dbd336b79148aa742883198bbf60342,115
+ethereum,bebop,1,2023-04-07,0xd41613c9af3ef4adf84279f2b211a441aa3a37dd1c546d018ca6082911ccf743,24,,0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2,0.08539524191852751,0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48,170.855956
```

Comment:
Hi @B1boid, you can fill in the `trace_address` field in this seed file as well.

Reply:
Thanks, done ✅